mobileapp/components/game/Game.tsx

```diff
@@ -2,6 +2,7 @@ import { useState, useEffect } from "react";
 import { View, ActivityIndicator, Button, Text } from "react-native";
 import { API_ENDPOINT } from "@env";
 import { Board, GameState } from "@kamon/core";
+import EventSource, { EventSourceListener } from "react-native-sse";
```

Comment:
`EventSourceListener` is useless here and `react-native-sse` seems to be not present in `packages.json` so the package is not installed